### ISSUE TITLE
Required package scikit-learn is not installed

### ISSUE THREAD

**Issue by tschager** [d3blocks/d3blocks#5]
D3Blocks requires sklearn, but the module is not listed in `setup.py`
If d3blocks is installed in an empty environment, it cannot be used due to a ModuleNotFoundError:
```
python -m venv venv
source ./venv/scripts/activate
pip install d3blocks
python -c 'import d3blocks'
> ...
> ModuleNotFoundError: No module named 'sklearn'
```

**Comment by erdogant** [comment]
Thanks for the feedback. I added sklearn to the setup and requirements file.
Update to the latest version with: `pip install -U d3blocks`

**Comment by tschager** [comment]
unfortunately not, but using scikit-learn instead of sklearn seems to work. I've created a pull request https://github.com/d3blocks/d3blocks/pull/6

**Comment by erdogant** [comment]
New version released (v1.1.4)
